### ISSUE TITLE
Wine can't delete connect settings, but windows 10 is normal behavior

### ISSUE THREAD

**Issue by jjasoncool** [HeidiSQL/HeidiSQL#1204]
**To Reproduce**
1. Go to works manager
2. delete any connection settings
3. exception occur.

**Expected behavior**
delete the settings

**Screenshots**
![2020-10-31 15-26-23 的螢幕擷圖](https://user-images.githubusercontent.com/24731867/97773920-be7b7500-1b8e-11eb-83d6-c69c7e0bfa0d.png)

**Environment:**
 - OS: fedora 33 wine 5.0
 - HeidiSQL version [11.0.0.6113]
 - Database type and version [MariaDB 10.5.5] 
I tested it is not having relation with the db version.

**Comment by ansgarbecker** [comment]
Please test whether this is happening as well with the default "Windows" theme. See Tools > Preferences > Style Theme. The location where the crash happens is suspicious for these themes.
In any case, please also attach the textfile with the whole bugreport in it (from the "save bug report" button)

**Comment by jjasoncool** [comment]
Ok, thank you

**Comment by jjasoncool** [comment]
[bugreport.txt](https://github.com/HeidiSQL/HeidiSQL/files/5489451/bugreport.txt)

**Comment by ansgarbecker** [comment]
Ok, the crash is not very talkative:

```
exception class    : EAccessViolation
exception message  : Access violation at address 000000006E7475D9 in module 'comctl32.dll'. Read of address 0000000000000008.

main thread ($2b):
6e7475d9 comctl32.dll
7bcc70d1 ntdll.dll       NtWaitForMultipleObjects
7b058744 kernelbase.dll  WaitForMultipleObjectsEx
7fd790d2 winex11.drv     MsgWaitForMultipleObjectsEx
7fd791ec user32.dll      GetMessageW
```

Please retry once, using the default "Windows" theme. Thanks!

**Comment by jjasoncool** [comment]
I can't change windows default theme, too.
[bugreport.txt](https://github.com/HeidiSQL/HeidiSQL/files/5506497/bugreport.txt)

**Comment by jjasoncool** [comment]
when I delete the protable_settings.txt
and add new connection, it will works.

So, can I using dark theme in wine?

**Comment by ansgarbecker** [comment]
I just tried that on Windows 10, in portable mode, with the Windows theme and once with the Material theme. But I get no such crash. It just works here.

**Comment by jjasoncool** [comment]
> I just tried that on Windows 10, in portable mode, with the Windows theme and once with the Material theme. But I get no such crash. It just works here.

This is fixed after update wine version to 6.0.0
